diff --git a/node_modules/pwa-kit-react-sdk/webpack/config.js b/node_modules/pwa-kit-react-sdk/webpack/config.js
index 66456dc..32d0e75 100644
--- a/node_modules/pwa-kit-react-sdk/webpack/config.js
+++ b/node_modules/pwa-kit-react-sdk/webpack/config.js
@@ -198,6 +198,9 @@ const common = {
       use: {
         loader: 'html-loader'
       }
+    }, {
+      test: /\.css$/i,
+      use: ['style-loader', 'css-loader']
     }]
   },
   externals
@@ -268,6 +271,9 @@ const ssrServerConfig = (0, _extends2.default)({}, {
     }, {
       test: /\.svg$/,
       loader: 'svg-sprite-loader'
+    }, {
+      test: /\.css$/i,
+      use: ['style-loader', 'css-loader']
     }]
   },
   stats
@@ -287,6 +293,9 @@ const requestProcessor = (0, _extends2.default)({}, {
       test: /\.js$/,
       exclude: /node_modules/,
       use: babelLoader
+    }, {
+      test: /\.css$/i,
+      use: ['style-loader', 'css-loader']
     }]
   },
   stats
